lab-examples/cvx03/lab-start.yml

```diff
@@ -15,14 +15,14 @@ topology:
       image: networkop/cx:4.3.0
 
     server01:
-      kind: cvx
+      kind: linux
```

Comment:
Hi @networkop 
what is the role of `kernel` key under the node config for a `linux` kind? is it because `runtime==ignite` is set via cmd flag there?

Reply:
actually, it's a leftover, should be removed. it's only relevant to ignite runtime nodes

Reply:
but then the image, is it a correct combination:

```yaml
server02:
      kind: linux
      image: networkop/host:ignite
```

Reply:
or should the change be `kind: linux->cvx`?

Reply:
yes, this image also works for both ignite and docker runtimes. the tag is just poor naming from me. I initially created it to run server nodes with ignite but then realised it works as well for non-ignite usecases.
You can see the buildfiles for it here https://github.com/networkop/cx/tree/main/host

Reply:
so there's nothing ignite-specific in it

Reply:
ok, will clean this up